### PR TITLE
Show Error on 'Mark as Sent' page for automatic-send partners

### PR DESCRIPTION
**Pull request by lalit97**
If a Partner's authorization method is Link, Proxy or Bundle
then the applications are sent automatically. So the 'Mark as Sent'
page is no longer needed for that Partner.

Bug: T234551

### CODE REVIEW

File: TWLight/applications/views.py [hunk 1]
```diff
@@ -1161,6 +1161,16 @@ class SendReadyApplicationsView(PartnerCoordinatorOnly, DetailView):
...
 
+    def dispatch(self, request, *args, **kwargs):
+        partner = self.get_object()
+        auth_method = partner.authorization_method
+        if auth_method == Partner.EMAIL or auth_method == Partner.CODES:
+            return super(SendReadyApplicationsView, self).dispatch(
+                request, *args, **kwargs
+            )
+        else:
+            raise Http404("Application's for this Partner are sent automatically")
```
Comment (Samwalton9) [inline TWLight/applications/views.py]: Just one small typo - "Application's" should be "Applications".

**Comment by Samwalton9** [comment]
This is looking great! There's just one typo that I've left a comment about, if you can fix that I'll get this merged :)

File: TWLight/applications/views.py (hunk 1 above)
Comment (lalit97) [reply TWLight/applications/views.py]: fixed it :)

**Comment by lalit97** [comment]
Thanks, I have fixed the typo :)

**Comment by Samwalton9** [comment]
Great, thanks!

**Pull request merged**